squid/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.7
```

Comment:
There may be a better place to put the `squid` directory than at the root, e.g. as a subdir of a `deps` or `external` folder.